### PR TITLE
Fix code layout and lint

### PR DESCRIPTION
**Pull request by cajus**

### CODE REVIEW

**Review by level420 — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Comment by level420** [comment]
2 approvals, more than 2 business days
merging.
thank you @cajus

File: framework/source/class/qx/ui/form/AbstractField.js [hunk 1]
```diff
@@ -935,31 +935,33 @@ qx.Class.define("qx.ui.form.AbstractField",
...
       }
     },
+
     /**
-     * validates the the input value
+     * Validates the the input value.
      * 
-     * @param {type} value: the value to check
-     * @returns the checked value
+     * @param value {Object} The value to check
```
Comment (schnepe2) [inline framework/source/class/qx/ui/form/AbstractField.js]: `{Object}` ? Really? This is like saying "anything". I am pretty sure this needs to be `{String|null}`

File: framework/source/class/qx/ui/form/AbstractField.js [hunk 2]
```diff
@@ -935,31 +935,33 @@ qx.Class.define("qx.ui.form.AbstractField",
...
+     * @returns The checked value
```
Comment (schnepe2) [inline framework/source/class/qx/ui/form/AbstractField.js]: `@returns` is not recognized, it's: `@return` (no "s")

**Comment by oetiker** [comment]
@schnepe2 comments "post-merge" don't help much ... cold you open a new PR to fix @returns

File: framework/source/class/qx/ui/form/AbstractField.js (hunk 1 above)
Comment (oetiker) [inline framework/source/class/qx/ui/form/AbstractField.js]: input can be literally anything ... so Object is appropriate

**Pull request merged**